Formula/luajit.rb

```diff
@@ -12,6 +12,10 @@ class Luajit < Formula
     sha256 "3ee1eeeca03770a402bc429aef0a09467f5b2efcdcb47c9a7d55355af0815c99" => :high_sierra
   end
 
+  stable do
+    patch :DATA
+  end
```

Comment:
```
  * C: 4: col 3: `url` should be put inside `stable` block
  * C: 5: col 3: `sha256` should be put inside `stable` block
  * C: 15: col 3: `stable` (line 15) should be put before `bottle` (line 8)
```

Move `url` and `sha256` into `stable` block. Move `stable` block above `bottle` block.

Reply:
Sorry, I misinterpreted the audit results: *If* there is a stable block, it needs to follow this rules.